### ISSUE TITLE
ExceptionInInitializerError with IntelliJ EAP build #IU-221.4501.155

### ISSUE THREAD

**Issue by JensVoss0253** [benwatson528/intellij-avro-parquet-plugin#88]
Running the plugin with the above IntelliJ version gives an error:

```
Unable to process file ... .parquet

java.lang.ExceptionInInitializerError
	at org.slf4j.impl.StaticLoggerBinder.<init>(StaticLoggerBinder.java:72)
	at org.slf4j.impl.StaticLoggerBinder.<clinit>(StaticLoggerBinder.java:45)
	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:417)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:362)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:388)
	at org.apache.hadoop.conf.Configuration.<clinit>(Configuration.java:227)
	at uk.co.hadoopathome.intellij.viewer.fileformat.ParquetFileReader.<init>(ParquetFileReader.java:52)
	at uk.co.hadoopathome.intellij.viewer.FileViewerToolWindow$2.doInBackground(FileViewerToolWindow.java:192)
	at uk.co.hadoopathome.intellij.viewer.FileViewerToolWindow$2.doInBackground(FileViewerToolWindow.java:184)
	at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalStateException: Detected both log4j-over-slf4j.jar AND bound slf4j-log4j12.jar on the class path, preempting StackOverflowError. See also http://www.slf4j.org/codes.html#log4jDelegationLoop for more details.
	at org.slf4j.impl.Log4jLoggerFactory.<clinit>(Log4jLoggerFactory.java:54)
	... 17 more

```

**Comment by benwatson528** [comment]
Hello, can you run any of the test Parquet files that are in this repo? https://github.com/benwatson528/intellij-avro-parquet-plugin/tree/master/src/test/resources/parquet.

**Comment by benwatson528** [comment]
This is fixed in version 3.0.0 of the plugin - look out for the update prompt in IntelliJ in the next day or two.

**Comment by benwatson528** [comment]
This update is now live.
